lib/SyTest/Identity/Server.pm

```diff
@@ -101,11 +98,15 @@ sub on_request
          $req->respond( HTTP::Response->new( 400, "Bad Request", [ Content_Length => 0 ] ) );
          return;
       }
-      my $token = $self->{expected_tokens}{ join "\0", $medium, $address, $sender, $room_id };
-      unless( defined $token ) {
-         $req->respond( HTTP::Response->new( 500, "Internal Server Error", [ Content_Length => 0 ] ) );
-         return;
-      }
+      my $token = Data::GUID->new->as_string;
```

Comment:
Hmm.. Does the wire format definitely require GUIDs here, or does just some unique string suffice? If so it might be easier just to serial-number them internally in the program rather than pulling in the new `Data::GUID` dependency.